python/samples.py

```diff
@@ -0,0 +1,61 @@
+from time import sleep
+import redis
+
+r = redis.Redis(host='localhost', port=6379, db=0)
+
+##Strings: New Options for the SET command
+#Demo of the GET option
+print(r.set('favoriteflavor', "Mint Choc Chip"))
+print(r.set('favoriteflavor', "Vanilla Bean", get=True))
+print(r.get('favoriteflavor'))
+
+#Demo of the EXAT option
```

Comment:
I think it would be good for each demo to `print` out which demo it is, so that the output is easier to read.